### PR TITLE
Update Ruby Central logo in Readme and footer

### PR DESCRIPTION
**Pull request by megantiu**
Hi! 👋  I'm on the Board of Ruby Central and we're in the process of updating our branding. In this PR, I've updated the Ruby Central logo in the README as well as in the footer.

Like in #2189, I ran `sponsors.png` through ImageOptim to maintain a small image size.

Here's what the new logo looks like in the footer:

Desktop
Screen Shot 2020-07-08 at 12 09 14 PM

Tablet
Screen Shot 2020-07-08 at 12 12 28 PM

Mobile
Screen Shot 2020-07-08 at 12 12 52 PM

---

I don't love how close the logo is to the "Supported by" text on tablet/mobile screens, but after playing around with the CSS and positioning it differently within `sponsors.png` itself, that was the best I could do. I'm certainly open to feedback/suggestions!

cc @evanphx

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The Ruby community's gem host.
 ## Support
 
 <a href="https://rubytogether.org/"><img src="https://rubytogether.org/images/rubies.svg" width=200></a>
-<a href="https://rubycentral.org/"><img src="https://gist.githubusercontent.com/sonalkr132/52a23481c0765b36ce1e909ba678c51a/raw/78e47c9ec77b690322040d4b9b8f460f58196182/Ruby-Central-Logo.png" width=160></a><br/>
+<a href="https://rubycentral.org/"><img src="https://user-images.githubusercontent.com/12753198/86944188-5ef7e600-c115-11ea-9ab0-6afd83655909.png" height=110></a><br/>
```
Comment (dwradcliffe) [inline README.md:13]: We can probably include this file too.

Comment (sonalkr132) [reply README.md:13]: yeah, I had used githubusercontent as a temporary fix. Either add it rubycentral.org or add it to our public folder.

Comment (megantiu) [reply README.md]: Thanks @dwradcliffe @sonalkr132, updated!

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ The Ruby community's gem host.
 ## Support
 
 <a href="https://rubytogether.org/"><img src="https://rubytogether.org/images/rubies.svg" width=200></a>
-<a href="https://rubycentral.org/"><img src="https://gist.githubusercontent.com/sonalkr132/52a23481c0765b36ce1e909ba678c51a/raw/78e47c9ec77b690322040d4b9b8f460f58196182/Ruby-Central-Logo.png" width=160></a><br/>
+<a href="https://rubycentral.org/"><img src="public/images/ruby_central_logo.png" height=110></a><br/>
```
Comment (dwradcliffe) [inline README.md:13]: I'm not sure that path is right

Comment (sonalkr132) [reply README.md:13]: Are you concerned about adding it public folder? Only another place we have images is vendor/assets. I didn't want to introduce a special location for GitHub repo only images.

Comment (sonalkr132) [reply README.md]: I was wrong, we already have `doc` folder with erd.svg. moved this to `doc`.

Comment (dwradcliffe) [reply README.md]: Oh I didn't realize this was just a markdown file I thought this was web page content and the path was wrong. 🤦 Doc folder seems fine though.

**Comment by sonalkr132** [comment]
Thank you @megantiu :sparkle:

**Comment by megantiu** [comment]
Thanks y'all! I appreciate the `doc/` find, @sonalkr132 😄

**Pull request merged**